src/handler/valueWriter.js

```diff
@@ -23,9 +23,9 @@ async function saveLocalValue(tab, key, subKey = "", value) {
   return false;
 }
 
-async function saveCookieValue(tab, key, subKey = "", value) {
+async function saveCookieValue(tab, key, subKey = "", domain, value) {
   try {
-    return await chromeSaveCookieValue(tab, key, subKey, value);
+    return await chromeSaveCookieValue(tab, key, subKey, domain, value);
```

Comment:
This `chromeSaveCookieValue` function is getting crowded in terms of args, so probably it would be easier to use having it grouped some of these by their context - tab, {context of cookie}:

```suggestion
    return await chromeSaveCookieValue(tab, {key, subKey, domain, value});
```

WDYT?

Reply:
At this point, I'm strongly inclined to include typescript in the project.
Making this kind of changes is starting to get a bit scary 😅

I'm a bit resistant to make this change in this PR because by changing this, I feel like we would need to change local and session storage as well, which are out of scope for this PR.
What do you reckon if we move this change to a different issue?